fake_switches/juniper_qfx_copper/juniper_qfx_copper_netconf_datastore.py

```diff
@@ -34,3 +35,12 @@ def parse_trunk_native_vlan(self, interface_node, port):
 
     def get_trunk_native_vlan_node(self, interface_node):
         return interface_node.xpath("native-vlan-id")
+
+    @staticmethod
+    def validate(configuration):
+        for port in configuration.ports:
+            if port_is_in_trunk_mode(port) and \
+                    (port.trunk_vlans is None or len(port.trunk_vlans) == 0):
+                raise FailingCommitResults([TrunkShouldHaveVlanMembers(interface=port.name),
+                                            ConfigurationCheckOutFailed()])
+        return JuniperNetconfDatastore.validate(configuration)
```

Comment:
A teacher once gave me 2 errors, one for not having a period and another one for the capital letter following the missing period... THat sounds just like that.